### PR TITLE
Update open source strategy

### PR DESCRIPTION
**Pull request by cornelius**
- Remove actions. Most of them are done and the open ones are tracked on GitHub.
- Add links to artifacts which now exist
- Update text to reflect current state
- Fix names

### CODE REVIEW

File: project/open-source-strategy.md [hunk 1]
```diff
@@ -54,54 +50,37 @@ Core and starting point for the open source project is the development of the so
...
+There are templates for [issues](https://github.com/dtr-org/unit-e/tree/master/.github/ISSUE_TEMPLATE) and [pull requests](https://github.com/dtr-org/unit-e/blob/master/.github/PULL_REQUEST_TEMPLATE.md) which help to make sure that the quality of new ones is high and required information is there. This is especially useful for new and external contributors but can also serve as a reminder and guideline for everybody.
 
 #### Architectural decisions
 
-Architectural decisions are taken and documented using the [Architecture Decision Record](https://github.com/dtr-org/unit-e-docs/tree/master/adrs) framework. See [ADR-1](https://github.com/dtr-org/unit-e-docs/blob/master/adrs/2018-08-16-ADR-1%20Adopt%20ADR.md) for more details.
-
-##### Actions
-
-* [Decide on scope and visibility of ADRs](https://github.com/dtr-org/unit-e-docs/issues/27)
+Architectural decisions are taken and documented using the [Architecture Decision Record](https://github.com/dtr-org/unit-e-project/tree/master/adrs) framework. See [ADR-1](https://github.com/dtr-org/unit-e-project/blob/master/adrs/2018-08-16-ADR-1%20Adopt%20ADR.md) for more details.
```
Comment (Ruteri) [inline project/open-source-strategy.md]: The link to ADR-1 seems to be broken (404 for me). Maybe it should be https://github.com/dtr-org/unit-e-project/blob/master/adrs/ADR-0001.md#adr-1-adopt-adr

**Review by Ruteri — COMMENTED** [review]
Aside for the one link, ACK 1f575ae42783d123b0b1b61da6109962f77266b2

File: project/open-source-strategy.md (hunk 1 above)
Comment (cornelius) [reply project/open-source-strategy.md]: Will fix it.

**Review by scravy — APPROVED** [review]
ACK

**Review by Gnappuraz — APPROVED** [review]
I tested all the links present, ACK https://github.com/dtr-org/unit-e-project/pull/117/commits/dc1f75aa146db4feeb7b8c77cbd60f93cf23d58d

**Review by Ruteri — APPROVED** [review]
ACK dc1f75aa146db4feeb7b8c77cbd60f93cf23d58d

**Comment by cornelius** [comment]
By the way, I extracted some of the more general resources to an independent list: https://github.com/cornelius/awesome-open-source

**Pull request merged**